### ISSUE TITLE
Дать возможность указывать путь установки зависимостей

### ISSUE THREAD

**Issue by khorevaa** [oscript-library/opm#96]
Сейчас такая возможность отсутсвует, что не позволяет установить зависимости в нужный каталг

**Comment by pallid** [comment]
хочется чтоб была возможность указать что зависимости надо поставить в oscript_modules устанавливаемого пакета

**Comment by EvilBeaver** [comment]
Да, это нужно

**Comment by nixel2007** [comment]
@pallid это может использоваться только в пакетах-приложениях, которых не так уж и много. плюс часть пакетов уже перепиливается под all-in-one вариант.
в вашем кейсе оно тоже поможет? у вас не библиотека, а именно приложение?

**Comment by pallid** [comment]
@nixel2007 да, приложения (os.web), а там используются пакеты не из oscript, а из oscript_modules вебдвижка или приложения

**Comment by nixel2007** [comment]
Посмотрите как собирается opm и vanessa-runner. Если вам это нужно для собственного приложения, то можно не ждать доработки опм, а сделать самому в своём манифесте

**Comment by pallid** [comment]
понял, т.е собирать пакет уже с зависимостями, спасибо

**Comment by khorevaa** [comment]
@pallid Еще я делал установку пакета с опеределением каталога установки зависимых пакетов в другой каталог пример gitsync https://github.com/oscript-library/gitsync/blob/develop/src/core/%D0%9A%D0%BB%D0%B0%D1%81%D1%81%D1%8B/%D0%A3%D0%BF%D1%80%D0%B0%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5%D0%9F%D0%BB%D0%B0%D0%B3%D0%B8%D0%BD%D0%B0%D0%BC%D0%B8.os#L128

**Comment by EvilBeaver** [comment]
@pallid в os.web должно автоматически подхватывать каталог oscript_modules, там конфиг-файл на в каталоге движка лежит, который в $workdir\oscript_modules нацеливает сразу

**Comment by EvilBeaver** [comment]
@pallid не понял фразу

**Comment by pallid** [comment]
os.web подхватывает каталог oscript_modules из директории проекта, по когда ставим приложение из пакета, его зависимости ставятся в lib oscriptа

**Comment by pallid** [comment]
или можно пописать чтоб os.web смотрел в либы oscriptа

**Comment by EvilBeaver** [comment]
Нужно просто перед запуском веб-проекта вызывать локальную установку

    opm install -l

Ставит локально в modules все зависимости проекта и проект готов к запуску.
